src/Network/Network/Models/PSServiceTagInformation.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Microsoft.WindowsAzure.Commands.Common.Attributes;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public class PSServiceTagInformation
+    {
+        [Ps1Xml(Label = "Name", Target = ViewControl.Table, Position = 1)]
+        public string Name { get; set; }
+
+        public string Id { get; set; }
+
+        public string ChangeNumber { get; set; }
```

Comment:
Can you add servicetagproperties here?

Reply:
@chandrasekarsrinivasan I've added `x-ms-client-flatten` in Swagger to move props from `properties` up one level, how it's done for other resources with nested properties. Was that wrong?

Reply:
Yeah, don't move it one level. This is not a top-level object like other properties in NRP.